be/src/olap/schema_change.cpp

```diff
@@ -1942,6 +1942,7 @@ Status SchemaChangeHandler::_do_process_alter_tablet_v2(const TAlterTabletReqV2&
             reader_context.is_vec = config::enable_vectorized_alter_table;
             reader_context.delete_bitmap = &base_tablet->tablet_meta()->delete_bitmap();
             reader_context.version = Version(0, end_version);
+            reader_context.use_local_file_cache = false;
```

Comment:
I think reader type is better than local file cache because it contains more info.